slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/DatabaseVersionDao.java

```diff
@@ -15,6 +15,8 @@
 import org.jdbi.v3.core.Handle;
 
 public class DatabaseVersionDao {
+  public static final int EXPECTED_DATABASE_VERSION = 10;
+  public static final int VALIDATOR_ENABLE_FLAG_VERSION = 10;
```

Comment:
Looks like this is only used in the tests? Although I see it's related to the EXPECTED_DATABASE_VERSION, I think here it could potentially be confused as a feature flag of some sort.

Reply:
Hmm.. maybe need a better name. Wanted to give a name to the validator enable flag migration. Any suggestions?

Reply:
I forgot to mention it but I was thinking it would be better scoped in the test packages rather than the Dao itself (then I guess the name wouldn't matter too much). 
I confused things by mentioning feature flags, it's more about the fact that I'd expect this constant to be used in production code if it's here.